sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 4.7.0-beta.1 (Unreleased)
+## 4.7.0 (Unreleased)
```

Comment:
See comments about this file in the previous CHANGELOG's review. Same applies for all CHANGELOGs.